src/dlgRoomExits.cpp

```diff
@@ -2286,13 +2302,14 @@ void dlgRoomExits::slot_checkModified()
                                            ? 1 : 0 ;
                     currentExit.weight = pI->text(2).toInt();
                     currentExit.hasStub = false;
-                    if( foundMap.contains( currentCmd )
-                        && foundMap.value( currentCmd )->destination == currentExit.destination
-                        && foundMap.value( currentCmd )->door        == currentExit.door
-                        && foundMap.value( currentCmd )->hasNoRoute  == currentExit.hasNoRoute
-                        && foundMap.value( currentCmd )->weight      == currentExit.weight      )
+                    auto exit = foundMap.value(currentCmd);
```

Comment:
clang-format's current settings don't result in this looking very nice, so my attempt here is to fix the coverity issue without dealing with the formatting. That will be dealt with later, so hopefully we can just have less coverity issues even if it doesn't look good yet.